lib/improver/tests/lapse_rate/Test_LapseRate.py

```diff
@@ -266,9 +288,41 @@ def test_constant_temp_orog(self):
                                                     self.land_sea_mask)
         self.assertArrayAlmostEqual(result.data, expected_out, decimal=4)
 
+    def test_fails_if_max_less_min_lapse_rate(self):
+        """Test code raises a Value Error if input maximum lapse rate is
+        less than input minimum lapse rate"""
+        msg = "Maximum lapse rate is less than minimum lapse rate"
+
+        with self.assertRaisesRegexp(ValueError, msg):
+            LapseRate(
+                max_lapse_rate=-1,
+                min_lapse_rate=1).process(self.temperature,
+                                          self.orography,
+                                          self.land_sea_mask)
+
+    def test_fails_if_nbhood_radius_less_than_zero(self):
+        """Test code raises a Value Error if input neighbourhood radius
+        is less than zero"""
+        msg = "Neighbourhood radius is less than zero"
+
+        with self.assertRaisesRegexp(ValueError, msg):
+            LapseRate(nbhood_radius=-1).process(self.temperature,
+                                                self.orography,
+                                                self.land_sea_mask)
+
+    def test_fails_if_max_height_diff_less_than_zero(self):
+        """Test code raises a Value Error if input neighbourhood radius
+        is less than zero"""
```

Comment:
Doc string needs changing to match test.

Reply:
Doc string changed.